tests/test_helpers.py

```diff
@@ -575,6 +575,9 @@
     (["helper", "f_todo", "--index", "99100"],
```

Comment:
the index and uuid should be handled in the same way as each other as we discussed, so we probably either need to change how index is handled (remove optional argument --index and take is a required argument) or have the uuid taken in --index.  I am not sure there is a way to test this unless we run the tests again from scratch.   Maybe it is time to set that up.

Given that choice I would be inclined to pass the uuid as index. Maybe we change that variable name to --index_uuid or something like that?  We will have to comment out the test.

Reply:
I would be inclined to change variable name to index_uuid or adding a variable name uuid. Because it would be clearer to the user what to enter. If we change it to index_uuid it is not as clear because one can think it means entering the index and uuid. If we leave it as just index, I will change the definition of described in index to add that it is interchangeable with the first 6 characters of a todo's uuid.

Reply:
My idea is that the user can enter either an index or a uuid.  Either would work in exactly the same way.  If I enter `regolith helper f_todo --index 123` or `regolith helper f_todo --index 8yt4q` will work the same way if todo with index 123 has uuid 8yt4q.

Let's just keep the arg name as index for now.  Late we can easily change it if we find a better name.